### PR TITLE
Add download links

### PR DESCRIPTION
**Pull request by honkfestival**
Now that COVID Alert is out, let's link to it!

@emilybulger @annemarieleger could you please help with a French translation?

cc @dfmcphee

### CODE REVIEW

**Review by dfmcphee — APPROVED** [review]
Code changes look good 👍

**Review by stonith — APPROVED** [review]

File: i18n/fr.yaml [hunk 1]
```diff
@@ -6,11 +6,11 @@ title:
   other: "COVID Shield"
 intro:
   other: "COVID Shield est une solution de notification d'exposition à la COVID-19 libre d'accès, conçue dans le respect absolu de la confidentialité. Elle a été créée par un groupe de bénévoles dans le but d'aider les Canadiens et le reste du monde à reprendre le travail dans de bonnes conditions de sécurité."
-disclaimer_title:
+download_title:
   other: "Comment puis-je télécharger et utiliser COVID Shield?"
-disclaimer_message:
+download_message:
   other: "COVID Shield n'est pas disponible actuellement pour usage, ni pour téléchargement."
```
Comment (emilybulger) [inline i18n/fr.yaml]: Change to

`Alerte COVID, la version canadienne officielle de COVID Shield, est maintenant disponible pour <a href="https://apps.apple.com/fr/app/id1520284227>iOS</a> et <a href="https://play.google.com/store/apps/details?id=ca.gc.hcsc.canada.stopcovid&hl=fr>Android</a>.`

**Review by emilybulger — CHANGES_REQUESTED** [review]
@honkfestival fr string added. Otherwise, looks good to me.

File: i18n/fr.yaml (hunk 1 above)
Comment (honkfestival) [reply i18n/fr.yaml]: All done!

**Comment by honkfestival** [comment]
@dfmcphee will this deploy automatically on merge?

**Comment by dfmcphee** [comment]
Yep, this should auto deploy once merged.

**Pull request merged**